pylint not load rcfile in tox

### Question

I do exactly same thing but cant get same result in Pure command and tox command.

I guess `MESSAGES CONTROL`  is not loading to setting.

How do I check rcfile setting is loading correctly?



command
pylint --verbose --rcfile=tox.ini src
```
Using config file C:\Users\pega_user\Desktop\vs_project\work3\serverapi\tox.ini

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

tox -r -e linting
```
...
src\server_api\__init__.py:40:4: C0415: Import outside toplevel (requests) (import-outside-toplevel)
src\server_api\__init__.py:40:4: E0401: Unable to import 'requests' (import-error)
src\server_api\__init__.py:44:11: W0703: Catching too general exception Exception (broad-except)
src\server_api\__init__.py:49:17: W1202: Use lazy % formatting in logging functions (logging-format-interpolation)
...
```

tox.ini
```
[testenv:linting]
description = linting test
# cant skip install
skip_install = True
deps = pylint
commands =
           # cmd /c dir
           pylint --rcfile=tox.ini src

[MESSAGES CONTROL]
disable=C,R,W
# no use now dont know why
[pylint]

# pylint --disable=all --enable=F,E,W
#* (C) convention, for programming standard violation
#* (R) refactor, for bad code smell
#* (W) warning, for python specific problems
#* (E) error, for probable bugs in the code
#* (F) fatal, if an error occurred which prevented pylint from doing further processing.

# never-returning-functions=sys.exit
max-line-length=150
#max-module-lines=5000
#max-nested-blocks=10
#min-public-methods=1
#max-public-methods=20
#max-statements=200
#max-branches=50
#max-locals=75
#max-args=10
#notes=
#    FIXME,
#    XXX,
#    TODO

#ignore-paths=^src/server_api/__main__.py$,


disable=
    #__main__.py,
    attribute-defined-outside-init,
    invalid-name,
    missing-docstring,
    protected-access,
    too-few-public-methods,
    # handled by black
    format,
    # We anticipate #3512 where it will become optional
    fixme,
    cyclic-import,
```





### Documentation for future user

How to check setting or Why I cant get same result between command and tox.

### Additional context

_No response_

--- a/doc/user_guide/usage/run.rst
+++ b/doc/user_guide/usage/run.rst
@@ -96,6 +96,8 @@ configuration file in the following order and uses the first one it finds:
    in on the command line.
 #. ``setup.cfg`` in the current working directory,
    providing it has at least one ``pylint.`` section
+#. ``tox.ini`` in the current working directory,
+   providing it has at least one ``pylint.`` section
 #. If the current working directory is in a Python package, Pylint searches \
    up the hierarchy of Python packages until it finds a ``pylintrc`` file. \
    This allows you to specify coding standards on a module-by-module \

